production/helm/loki/distributed-values.yaml

```diff
@@ -30,6 +30,7 @@ deploymentMode: Distributed
 
 ingester:
   replicas: 3
+  maxUnavailable: 2
```

Comment:
Both `maxUnavailable: 2` an `maxUnavailable: 1` can make deployment succeed, please let me know which one is better, thanks! ☺️